src/amp-events.js

```diff
@@ -21,6 +21,7 @@
 export const AmpEvents = {
   BUILT: 'amp:built',
   DOM_UPDATE: 'amp:dom-update',
+  VALUE_CHANGE: 'amp:value-change',
```

Comment:
How do we plan on using this event? Is the situation in this PR the only situation we would like to use it?

The reason why I am asking is this feels more like "amp-form input value change" event only from this PR.

Reply:
I think it's best if this event is only dispatched for form field value changes.

`FORM_VALUE_CHANGE: 'amp:form-value-change'`? Not sure if "input" would be ambiguous since `<input>` element vs general user input fields. I don't feel strongly about either tho.

Reply:
FORM_VALUE_CHANGE SG